src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -1237,10 +1250,10 @@ private void AnalyzeScenarios(int[] scenarios)
 
                 testCase.FileSystem = null;
                 testCase.Files = multiThreadTargets;
-                Run runMultiThread = RunAnalyzeCommand(options, testCase, multithreaded: true);
+                Run runMultithreaded = RunAnalyzeCommand(options, testCase, multithreaded: true);
```

Comment:
>runMultithreaded [](http://example.com/codeflow?start=20&length=16)

Notice that I fix incidental code issues that I notice while I'm working. My goal is that *every* code change leaves the code base in a slightly better condition, even when I am extending/adding functionality to it.